lib/stimulus_reflex/morph_mode/selector_morph_mode.rb

```diff
@@ -0,0 +1,15 @@
+module StimulusReflex
+  class SelectorMorphMode < MorphMode
+    def broadcast(reflex, selectors, data)
+      broadcast_message subject: "selector", data: data
```

Comment:
Shouldn't this be a call to `broadcast_morphs`?

Reply:
This was literally copy pasted. Maybe @leastbad has some insights?

Reply:
I do! 🤠

The actual changes are broadcast from reflex.rb; this triggers the callbacks.

Selector Morphs will broadcast N+1 times.